.github/workflows/build_and_test.yml

```diff
@@ -99,6 +99,14 @@ jobs:
           path: |
             rskj-core/build/reports/
 
+      - name: Upload HTML Test Report
+        if: failure()
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 #v4.6.2
+        with:
+          name: smell-test-html-test-report
+          path: |
+            rskj-core/build/reports/
```

Comment:
Just a small comment about these files, How many days will these files be available?
I believe the default is 90. If that's correct, we could reduce it using the` retention-days `property to avoid overloading the storage with unnecessary files.

Reply:
That makes sense, do you know which would be correct? I'm not very familiar to the  Github Action rules

Reply:
I believe that one week should be enough. The value should be useful for the devs, not for us. Nevertheless, it's GitHub's storage, not ours, so _we_ don't face issues with the storage use. :D

Reply:
WIll add retention-days of one week (7 days) then